axlearn/common/config.py

```diff
@@ -394,6 +394,49 @@ def set(self, **kwargs):
             setattr(self, k, v)
         return self
 
+    def get_recursively(self, path: Sequence[str]) -> Any:
+        """Recursively find the target key in the config and return its value.
+
+        Args:
+            path: A sequence of keys for indexing to get the target value.
+
+        Raises:
+            AttributeError: If key in path is not found.
+
+        Returns:
+            value at the path or self if path is empty.
+        """
+        current = self
+        index = 0
+
+        while path and index < len(path):
+            key = path[index]
+
+            # TODO(markblee): maybe use cfg.visit instead of getattr
+            current = getattr(current, key)
+            index += 1
+
+            if index == len(path):
+                break
```

Comment:
Hm, did I miss something or can we just do:
```suggestion
        for part in path:
            current = getattr(current, part)
```

(In any case, I can make some changes in a follow-up.)

Reply:
Made this change, thanks @markblee